<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleplayerDialog</class>
 <widget class="QDialog" name="singleplayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Singleplayer Kniffel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QTableWidget" name="tW_SpielstandSingle">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <row>
      <property name="text">
       <string>Einsen</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Zweien</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Dreien</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Vieren</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Fünfen</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Sechsen</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Dreierpasch</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Viererpasch</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Full House</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kleine Straße</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Große Straße</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Kniffel</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Chance</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Du</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>KI</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="4">
      <widget class="QPushButton" name="pBW5">
       <property name="text">
        <string>behalten</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pBW1">
       <property name="text">
        <string>behalten</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pBW4">
       <property name="text">
        <string>behalten</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pBW3">
       <property name="text">
        <string>behalten</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pBW2">
       <property name="text">
        <string>behalten</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pBwuerfeln">
     <property name="text">
      <string>Würfeln</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber_1"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
     <item row="0" column="4">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>singleplayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>singleplayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
